layers/+writing/vimwiki/config.vim

```diff
@@ -0,0 +1,3 @@
+" goyo.vim {
+  nnoremap <Leader>Ww <Plug>:VimwikiIndex<CR>
```

Comment:
Why not `<Leader>wW`? That should be easier to type.

Reply:
done